recipe/meta.yaml

```diff
@@ -37,6 +38,8 @@ requirements:
     - cython >=0.29,<0.30
     - spdlog >=1.8.5,<2
     - cuda-python 11.6
+    - thrust 1.15.*
+    - cub 1.15.*
```

Comment:
I'm not sure if these are necessary or if the librmm package contains its own copy of thrust and cub for rmm Python to include and use. Let's see if this works first, but if it does I would try to remove them just in case.

Reply:
Right in the previous builds these were needed because otherwise CMake confuses itself with the copies from CTK, and we got a (Thrust/CUB) version mismatch error. Looks like we can remove them after switching to scikit-build. Will give it a shot.